# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2024/12
# use bazel binary from module, if loaded.
diff -ru tensorstore-0.1.69/bazelisk.py tensorstore-0.1.69_disable_downloads/bazelisk.py
--- tensorstore-0.1.69/bazelisk.py	2024-11-25 23:54:46.000000000 +0100
+++ tensorstore-0.1.69_disable_downloads/bazelisk.py	2024-12-12 17:09:45.116084719 +0100
@@ -487,8 +487,11 @@
 def main(argv=None):
     if argv is None:
         argv = sys.argv
-
-    bazel_path = get_bazel_path()
+    bazel_path = ''
+    if (os.getenv('EBROOTBAZEL')): 
+        bazel_path=os.path.join(os.getenv('EBROOTBAZEL'), 'bin', 'bazel')
+    else:
+        bazel_path = get_bazel_path()
 
     argv = argv[1:]
 
